### ISSUE TITLE
Introduce wildcard field in GraphQL to retrieve the contentlet's map of properties

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#20203]
```
query PageAPI {
  page(url: "/index") {
    title
    url
    seodescription
    titleImage{
      name
    }
    host {
      hostName
    }
    layout {
      header
      footer
      body {
        rows {
          columns {
            containers {
              identifier
              uuid
              contentlets{
                  */*/*.   <---- Wildcard that specifies depth
              }
              rendered{
              <---- rendered container
              }
              
            }
          }
        }
      }
    }
  }
}
```

**Comment by dsilvam** [comment]
Note to QA: For a sample usage see postman test `GraphQL.PageAPI_TestMapSpecialField`

**Comment by victoralfaro-dotcms** [comment]
Results with just providing `_map` GrpahQL special field:
```
"contentlets": [{
	"_map": {
		"myFile": {
			"identifier": "8dd88708-c600-4366-8a50-9eec2af9091b",
			"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
			"extension": "jpg",
			"isContentlet": true,
			"description": "isalnds-french-polynesia",
			"__icon__": "jpgIcon",
			"mimeType": "image/jpeg",
			"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
			"type": "file_asset",
			"metaData": {
				"path": "c/5/c5ad88f9-31b3-458b-87ed-64f2c1cf82b5/fileAsset/isalnds-french-polynesia.jpg",
				"isImage": true,
				"modDate": 1621890340000,
				"sha256": "6c6658a841c3ae9a60d20143d4de6e320664411a9458b92c787b6201136164e5",
				"fileSize": 102816,
				"length": 102816,
				"name": "isalnds-french-polynesia.jpg",
				"width": 780,
				"title": "isalnds-french-polynesia.jpg",
				"contentType": "image/jpeg",
				"height": 520
			},
			"path": "/images/blogs/",
			"size": 102816,
			"name": "isalnds-french-polynesia.jpg",
			"width": 780,
			"underlyingFileName": "isalnds-french-polynesia.jpg",
			"height": 520
		},
		"publishDate": 1623348949091,
		"language": {
			"languageId": 1,
			"language": "English",
			"languageCode": "en",
			"country": "United States",
			"countryCode": "US",
			"languageFlag": "en_US",
			"isoCode": "en-us",
			"id": 1
		},
		"postingDate": 1562955300000,
		"body": "<p>Curabitur egestas mi eget massa maximus, eget dapibus ex feugiat. Vestibulum a dapibus tellus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Etiam lorem orci, pellentesque quis leo at, dapibus elementum metus.</p>\n<p>Nullam in sollicitudin nibh. Aenean eget est ac quam facilisis placerat ac vitae diam. Suspendisse est sem, fermentum nec massa sit habitasse platea dictumst. Maecenas tellus sem, imperdiet Mauris aliquam ipsum et mi pulvinar, sed iaculis nunc tristique. Etiam aliquam velit orci, vitae dignissim turpis elementum quis. Phasellus eu enim lobortis, pellentesque erat quis, elementum nisl. Praesent vehicula ex eu libero fermentum sodales. Pellentesque facilisis gravida nibh id tempor. Vivamus elementum sit amet leo non iaculis. Proin id magna convallis, porta tortor in, consequat enim. Curabitur in massa justo. vel ante sed, consectetur auctor diam. Vivamus aliquam nisl non.</p>\n<blockquote>\n<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean dictum venenatis nulla, quis porta sem pellentesque rutrum. Curabitur ullamcorper lacus ac sem pulvinar placerat. Sed leo ipsum, convallis ac erat nec, porta efficitur neque. Sed non elementum eros, id maximus lorem. Integer auctor lectus quis ultricies dictum. Curabitur egestas mi eget dapibus ex feugiat.</p>\n</blockquote>\n<p>Pellentesque quis leo at, dapibus elementum metus. Aenean molestie aliquam ex, in dignissim odio lobortis nec. Nam blandit metus tincidunt, mattis velit vitae, viverra lorem. Vivamus venenatis nunc nisi, vitae facilisis est porttitor vitae. Donec et dui vulputate, finibus eros sed, ultrices mauris. Donec eleifend risus arcu. Maecenas massa ex, hendrerit ac arcu ac, congue rhoncus neque. Duis ut fermentum lacus, non dapibus turpis. Nunc vel tempor erat. Quisque gravida iaculis lacus at faucibus.</p>",
		"inode": "1f4db79c-7ec6-4efe-9369-1ae2193a2d7f",
		"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
		"locked": false,
		"stInode": "799f176a-d32e-4844-a07c-1b5fcd107578",
		"contentType": "Blog",
		"identifier": "0edfee78-2a75-4f3d-bf20-813aae15d4e9",
		"image": {
			"modDate": 1621890340000,
			"sha256": "213c68196704f47b603972b21054c756ef89dde9b76de5287d3d5f5f3d3af225",
			"mime": "image/jpeg",
			"title": "gallery-7-1200x800-original.jpg",
			"versionPath": "/dA/1f4db79c7e/image/gallery-7-1200x800-original.jpg",
			"focalPoint": "0.0",
			"path": "1/f/1f4db79c-7ec6-4efe-9369-1ae2193a2d7f/image/gallery-7-1200x800-original.jpg",
			"isImage": true,
			"idPath": "/dA/0edfee782a/image/gallery-7-1200x800-original.jpg",
			"size": 1098821,
			"name": "gallery-7-1200x800-original.jpg",
			"width": 0,
			"height": 0
		},
		"urlTitle": "10-new-snow-sports-to-try-this-winter",
		"author": [
			"f851e34a-d128-408e-a75a-4fa1c33201b8"
		],
		"myMap": "myMapValue",
		"tags": "snowboarding,winter,snowmobiling,snow sports,skiing,cross-country skiing",
		"myCat": {
			"categories": [{
					"inode": "14ab7eca-7470-4f01-9671-a61f535c9b3a",
					"active": true,
					"name": "Apparel",
					"key": "apparel",
					"keywords": "",
					"velocityVar": "apparel"
				},
				{
					"inode": "4c32d127-4f97-4db8-a1c8-c8934cc2b034",
					"active": true,
					"name": "Outdoor",
					"key": "outdoor",
					"keywords": "",
					"velocityVar": "outdoor"
				},
				{
					"inode": "c1075aae-5392-4272-94ae-1c3ae01a6d86",
					"active": true,
					"name": "Snow",
					"key": "snow",
					"keywords": "",
					"velocityVar": "snow"
				}
			]
		},
		"folder": "SYSTEM_FOLDER",
		"myImage": {
			"identifier": "4068c24e-4f15-472e-86fc-a57dc785f0c6",
			"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
			"extension": "jpg",
			"isContentlet": true,
			"description": "shark-feeding",
			"__icon__": "jpgIcon",
			"mimeType": "image/jpeg",
			"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
			"type": "file_asset",
			"metaData": {
				"path": "6/1/615cf6c7-a5fb-4c2d-bc0d-f27ec69d9118/fileAsset/shark-feeding.jpg",
				"isImage": true,
				"modDate": 1621890339000,
				"sha256": "03fc29afd486f484b22d5b19408d61bde01314d5164f36f02bacb0d140780065",
				"fileSize": 98885,
				"length": 98885,
				"name": "shark-feeding.jpg",
				"width": 780,
				"title": "shark-feeding.jpg",
				"contentType": "image/jpeg",
				"height": 413
			},
			"path": "/images/blogs/",
			"size": 98885,
			"name": "shark-feeding.jpg",
			"width": 780,
			"underlyingFileName": "shark-feeding.jpg",
			"height": 413
		},
		"hasTitleImage": true,
		"sortOrder": 0,
		"hostName": "demo.dotcms.com",
		"modDate": 1623348949091,
		"title": "10 New Snow Sports to Try This Winter",
		"baseType": "CONTENT",
		"myKeyValue": {
			"key1": "value1",
			"key2": "value2"
		},
		"archived": false,
		"working": true,
		"map": "mapValue",
		"live": true,
		"owner": "dotcms.org.1",
		"imageVersion": "/dA/1f4db79c-7ec6-4efe-9369-1ae2193a2d7f/image/gallery-7-1200x800-original.jpg",
		"imageContentAsset": "0edfee78-2a75-4f3d-bf20-813aae15d4e9/image",
		"languageId": 1,
		"URL_MAP_FOR_CONTENT": "/blog/post/10-new-snow-sports-to-try-this-winter",
		"url": "/content.1f4db79c-7ec6-4efe-9369-1ae2193a2d7f",
		"titleImage": "image",
		"modUserName": "Admin User",
		"urlMap": "/blog/post/10-new-snow-sports-to-try-this-winter",
		"hasLiveVersion": true,
		"blogComment": [
			"75e62bcd-294b-4f15-9d37-133c19799a18",
			"23edb4fa-a363-4dfa-b12b-8f3a07415248"
		],
		"modUser": "dotcms.org.1",
		"teaser": "$date - From the tame to the exotic, these alternative winter sports will keep you outside and in shape until all the snow"
	}
}]
```

**Comment by victoralfaro-dotcms** [comment]
`myFile`, `myImage` and `Language` showed as composed types:
```
"myFile": {
	"identifier": "8dd88708-c600-4366-8a50-9eec2af9091b",
	"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
	"extension": "jpg",
	"isContentlet": true,
	"description": "isalnds-french-polynesia",
	"__icon__": "jpgIcon",
	"mimeType": "image/jpeg",
	"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
	"type": "file_asset",
	"metaData": {
		"path": "c/5/c5ad88f9-31b3-458b-87ed-64f2c1cf82b5/fileAsset/isalnds-french-polynesia.jpg",
		"isImage": true,
		"modDate": 1621890340000,
		"sha256": "6c6658a841c3ae9a60d20143d4de6e320664411a9458b92c787b6201136164e5",
		"fileSize": 102816,
		"length": 102816,
		"name": "isalnds-french-polynesia.jpg",
		"width": 780,
		"title": "isalnds-french-polynesia.jpg",
		"contentType": "image/jpeg",
		"height": 520
	},
	"path": "/images/blogs/",
	"size": 102816,
	"name": "isalnds-french-polynesia.jpg",
	"width": 780,
	"underlyingFileName": "isalnds-french-polynesia.jpg",
	"height": 520
}
```
```
"myImage": {
	"identifier": "4068c24e-4f15-472e-86fc-a57dc785f0c6",
	"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
	"extension": "jpg",
	"isContentlet": true,
	"description": "shark-feeding",
	"__icon__": "jpgIcon",
	"mimeType": "image/jpeg",
	"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
	"type": "file_asset",
	"metaData": {
		"path": "6/1/615cf6c7-a5fb-4c2d-bc0d-f27ec69d9118/fileAsset/shark-feeding.jpg",
		"isImage": true,
		"modDate": 1621890339000,
		"sha256": "03fc29afd486f484b22d5b19408d61bde01314d5164f36f02bacb0d140780065",
		"fileSize": 98885,
		"length": 98885,
		"name": "shark-feeding.jpg",
		"width": 780,
		"title": "shark-feeding.jpg",
		"contentType": "image/jpeg",
		"height": 413
	},
	"path": "/images/blogs/",
	"size": 98885,
	"name": "shark-feeding.jpg",
	"width": 780,
	"underlyingFileName": "shark-feeding.jpg",
	"height": 413
}
```
```
"language": {
	"languageId": 1,
	"language": "English",
	"languageCode": "en",
	"country": "United States",
	"countryCode": "US",
	"languageFlag": "en_US",
	"isoCode": "en-us",
	"id": 1
}
```

**Comment by victoralfaro-dotcms** [comment]
`myFile`, `myImage` and `Language` showed as composed types:
```
"myFile": {
	"identifier": "8dd88708-c600-4366-8a50-9eec2af9091b",
	"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
	"extension": "jpg",
	"isContentlet": true,
	"description": "isalnds-french-polynesia",
	"__icon__": "jpgIcon",
	"mimeType": "image/jpeg",
	"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
	"type": "file_asset",
	"metaData": {
		"path": "c/5/c5ad88f9-31b3-458b-87ed-64f2c1cf82b5/fileAsset/isalnds-french-polynesia.jpg",
		"isImage": true,
		"modDate": 1621890340000,
		"sha256": "6c6658a841c3ae9a60d20143d4de6e320664411a9458b92c787b6201136164e5",
		"fileSize": 102816,
		"length": 102816,
		"name": "isalnds-french-polynesia.jpg",
		"width": 780,
		"title": "isalnds-french-polynesia.jpg",
		"contentType": "image/jpeg",
		"height": 520
	},
	"path": "/images/blogs/",
	"size": 102816,
	"name": "isalnds-french-polynesia.jpg",
	"width": 780,
	"underlyingFileName": "isalnds-french-polynesia.jpg",
	"height": 520
}
```
```
"myImage": {
	"identifier": "4068c24e-4f15-472e-86fc-a57dc785f0c6",
	"parent": "9716b18a-8b2f-44de-8c10-13f886c6f119",
	"extension": "jpg",
	"isContentlet": true,
	"description": "shark-feeding",
	"__icon__": "jpgIcon",
	"mimeType": "image/jpeg",
	"statusIcons": "<span class='greyDotIcon' style='opacity:.4'></span><span class='liveIcon'></span>",
	"type": "file_asset",
	"metaData": {
		"path": "6/1/615cf6c7-a5fb-4c2d-bc0d-f27ec69d9118/fileAsset/shark-feeding.jpg",
		"isImage": true,
		"modDate": 1621890339000,
		"sha256": "03fc29afd486f484b22d5b19408d61bde01314d5164f36f02bacb0d140780065",
		"fileSize": 98885,
		"length": 98885,
		"name": "shark-feeding.jpg",
		"width": 780,
		"title": "shark-feeding.jpg",
		"contentType": "image/jpeg",
		"height": 413
	},
	"path": "/images/blogs/",
	"size": 98885,
	"name": "shark-feeding.jpg",
	"width": 780,
	"underlyingFileName": "shark-feeding.jpg",
	"height": 413
}
```
```
"language": {
	"languageId": 1,
	"language": "English",
	"languageCode": "en",
	"country": "United States",
	"countryCode": "US",
	"languageFlag": "en_US",
	"isoCode": "en-us",
	"id": 1
}
```

**Comment by victoralfaro-dotcms** [comment]
With `depth: 1`
```
"author": [{
	"publishDate": "2021-06-02 12:38:07.685",
	"language": {
		"languageId": 1,
		"language": "English",
		"languageCode": "en",
		"country": "United States",
		"countryCode": "US",
		"languageFlag": "en_US",
		"isoCode": "en-us",
		"id": 1
	},
	"inode": "baf596fa-7592-4336-8674-b2ef6d8af748",
	"profilePhoto": {
		"modDate": 1621890340000,
		"sha256": "b1aa20d9c5420987d2503db6eea29074f478e6b63d73dbf6f48f8db39448303b",
		"mime": "image/jpeg",
		"title": "john-smith.jpg",
		"versionPath": "/dA/baf596fa75/profilePhoto/john-smith.jpg",
		"focalPoint": "0.0",
		"path": "b/a/baf596fa-7592-4336-8674-b2ef6d8af748/profilePhoto/john-smith.jpg",
		"isImage": true,
		"idPath": "/dA/f851e34ad1/profilePhoto/john-smith.jpg",
		"size": 2800,
		"name": "john-smith.jpg",
		"width": 0,
		"height": 0
	},
	"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
	"locked": false,
	"stInode": "045d0b52-fa68-465a-b820-f9fc22febc50",
	"contentType": "BlogAuthor",
	"identifier": "f851e34a-d128-408e-a75a-4fa1c33201b8",
	"firstName": "John",
	"folder": "SYSTEM_FOLDER",
	"hasTitleImage": true,
	"sortOrder": 0,
	"lastName": "Smith",
	"hostName": "demo.dotcms.com",
	"modDate": "2021-06-02 12:38:07.685",
	"title": "John",
	"baseType": "CONTENT",
	"archived": false,
	"working": true,
	"live": true,
	"owner": "036fd43a-6d98-46e0-b22e-bae02cb86f0c",
	"languageId": 1,
	"textFieldMap": "$date",
	"userId": "036fd43a-6d98-46e0-b22e-bae02cb86f0c",
	"url": "/content.baf596fa-7592-4336-8674-b2ef6d8af748",
	"titleImage": "profilePhoto",
	"modUserName": "Admin User",
	"hasLiveVersion": true,
	"modUser": "dotcms.org.1"
}]

...

"blogComment": [{
		"publishDate": "2021-06-18 11:27:18.565",
		"language": {
			"languageId": 1,
			"language": "English",
			"languageCode": "en",
			"country": "United States",
			"countryCode": "US",
			"languageFlag": "en_US",
			"isoCode": "en-us",
			"id": 1
		},
		"body": "new blog comment 1",
		"inode": "0a987cec-6cd1-4530-9fe0-bb74a8860ae3",
		"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
		"locked": false,
		"stInode": "6044a806-f462-4977-a353-57539eac2a2c",
		"contentType": "BlogComment",
		"identifier": "75e62bcd-294b-4f15-9d37-133c19799a18",
		"folder": "SYSTEM_FOLDER",
		"hasTitleImage": false,
		"sortOrder": 0,
		"hostName": "demo.dotcms.com",
		"modDate": "2021-06-18 11:27:18.565",
		"title": "new blog comment 1",
		"baseType": "CONTENT",
		"archived": false,
		"working": true,
		"live": true,
		"owner": "dotcms.org.1",
		"languageId": 1,
		"url": "/content.c18319c8-cf3f-49c1-86e2-f2c002adb84a",
		"titleImage": "TITLE_IMAGE_NOT_FOUND",
		"modUserName": "Admin User",
		"hasLiveVersion": true,
		"modUser": "dotcms.org.1"
	}
]
```

**Comment by victoralfaro-dotcms** [comment]
With `depth: 2`
```
"blogComment": [{
		"publishDate": "2021-06-18 11:27:18.565",
		"language": {
			"languageId": 1,
			"language": "English",
			"languageCode": "en",
			"country": "United States",
			"countryCode": "US",
			"languageFlag": "en_US",
			"isoCode": "en-us",
			"id": 1
		},
		"body": "new blog comment 1",
		"inode": "0a987cec-6cd1-4530-9fe0-bb74a8860ae3",
		"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
		"locked": false,
		"stInode": "6044a806-f462-4977-a353-57539eac2a2c",
		"contentType": "BlogComment",
		"identifier": "75e62bcd-294b-4f15-9d37-133c19799a18",
		"folder": "SYSTEM_FOLDER",
		"hasTitleImage": false,
		"sortOrder": 0,
		"hostName": "demo.dotcms.com",
		"modDate": "2021-06-18 11:27:18.565",
		"title": "new blog comment 1",
		"baseType": "CONTENT",
		"archived": false,
		"working": true,
		"live": true,
		"owner": "dotcms.org.1",
		"languageId": 1,
		"url": "/content.c18319c8-cf3f-49c1-86e2-f2c002adb84a",
		"titleImage": "TITLE_IMAGE_NOT_FOUND",
		"modUserName": "Admin User",
		"hasLiveVersion": true,
		"modUser": "dotcms.org.1",
		"commentAuthor": "4150dc1a-92c1-45b8-89ea-6b5c7bdedf74",
		"reply": []
	}
]
```

**Comment by victoralfaro-dotcms** [comment]
With `depth: 3`
```
"blogComment": [{
		"publishDate": "2021-06-18 11:27:18.565",
		"language": {
			"languageId": 1,
			"language": "English",
			"languageCode": "en",
			"country": "United States",
			"countryCode": "US",
			"languageFlag": "en_US",
			"isoCode": "en-us",
			"id": 1
		},
		"body": "new blog comment 1",
		"inode": "0a987cec-6cd1-4530-9fe0-bb74a8860ae3",
		"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
		"locked": false,
		"stInode": "6044a806-f462-4977-a353-57539eac2a2c",
		"contentType": "BlogComment",
		"identifier": "75e62bcd-294b-4f15-9d37-133c19799a18",
		"folder": "SYSTEM_FOLDER",
		"hasTitleImage": false,
		"sortOrder": 0,
		"hostName": "demo.dotcms.com",
		"modDate": "2021-06-18 11:27:18.565",
		"title": "new blog comment 1",
		"baseType": "CONTENT",
		"archived": false,
		"working": true,
		"live": true,
		"owner": "dotcms.org.1",
		"languageId": 1,
		"url": "/content.c18319c8-cf3f-49c1-86e2-f2c002adb84a",
		"titleImage": "TITLE_IMAGE_NOT_FOUND",
		"modUserName": "Admin User",
		"hasLiveVersion": true,
		"modUser": "dotcms.org.1",
		"commentAuthor": {
			"publishDate": "2021-06-18 11:27:01.32",
			"language": {
				"languageId": 1,
				"language": "English",
				"languageCode": "en",
				"country": "United States",
				"countryCode": "US",
				"languageFlag": "en_US",
				"isoCode": "en-us",
				"id": 1
			},
			"inode": "2310d67d-5f05-4253-b6bf-3c29f8204672",
			"profilePhoto": {},
			"host": "SYSTEM_HOST",
			"locked": false,
			"stInode": "fe719cc1-26d1-499f-8702-c6d35c661286",
			"contentType": "BlogCommentor",
			"identifier": "4150dc1a-92c1-45b8-89ea-6b5c7bdedf74",
			"firstName": "Victor",
			"folder": "SYSTEM_FOLDER",
			"hasTitleImage": false,
			"sortOrder": 0,
			"lastName": "Alfaro",
			"hostName": "System Host",
			"modDate": "2021-06-18 11:27:01.32",
			"title": "Victor",
			"baseType": "CONTENT",
			"archived": false,
			"working": true,
			"live": true,
			"owner": "dotcms.org.1",
			"languageId": 1,
			"url": "/content.2310d67d-5f05-4253-b6bf-3c29f8204672",
			"titleImage": "TITLE_IMAGE_NOT_FOUND",
			"modUserName": "Admin User",
			"hasLiveVersion": true,
			"modUser": "dotcms.org.1"
		},
		"reply": []
	}
]
```

**Comment by victoralfaro-dotcms** [comment]
With `depth: 3`
```
"blogComment": [{
		"publishDate": "2021-06-18 11:27:18.565",
		"language": {
			"languageId": 1,
			"language": "English",
			"languageCode": "en",
			"country": "United States",
			"countryCode": "US",
			"languageFlag": "en_US",
			"isoCode": "en-us",
			"id": 1
		},
		"body": "new blog comment 1",
		"inode": "0a987cec-6cd1-4530-9fe0-bb74a8860ae3",
		"host": "48190c8c-42c4-46af-8d1a-0cd5db894797",
		"locked": false,
		"stInode": "6044a806-f462-4977-a353-57539eac2a2c",
		"contentType": "BlogComment",
		"identifier": "75e62bcd-294b-4f15-9d37-133c19799a18",
		"folder": "SYSTEM_FOLDER",
		"hasTitleImage": false,
		"sortOrder": 0,
		"hostName": "demo.dotcms.com",
		"modDate": "2021-06-18 11:27:18.565",
		"title": "new blog comment 1",
		"baseType": "CONTENT",
		"archived": false,
		"working": true,
		"live": true,
		"owner": "dotcms.org.1",
		"languageId": 1,
		"url": "/content.c18319c8-cf3f-49c1-86e2-f2c002adb84a",
		"titleImage": "TITLE_IMAGE_NOT_FOUND",
		"modUserName": "Admin User",
		"hasLiveVersion": true,
		"modUser": "dotcms.org.1",
		"commentAuthor": {
			"publishDate": "2021-06-18 11:27:01.32",
			"language": {
				"languageId": 1,
				"language": "English",
				"languageCode": "en",
				"country": "United States",
				"countryCode": "US",
				"languageFlag": "en_US",
				"isoCode": "en-us",
				"id": 1
			},
			"inode": "2310d67d-5f05-4253-b6bf-3c29f8204672",
			"profilePhoto": {},
			"host": "SYSTEM_HOST",
			"locked": false,
			"stInode": "fe719cc1-26d1-499f-8702-c6d35c661286",
			"contentType": "BlogCommentor",
			"identifier": "4150dc1a-92c1-45b8-89ea-6b5c7bdedf74",
			"firstName": "Victor",
			"folder": "SYSTEM_FOLDER",
			"hasTitleImage": false,
			"sortOrder": 0,
			"lastName": "Alfaro",
			"hostName": "System Host",
			"modDate": "2021-06-18 11:27:01.32",
			"title": "Victor",
			"baseType": "CONTENT",
			"archived": false,
			"working": true,
			"live": true,
			"owner": "dotcms.org.1",
			"languageId": 1,
			"url": "/content.2310d67d-5f05-4253-b6bf-3c29f8204672",
			"titleImage": "TITLE_IMAGE_NOT_FOUND",
			"modUserName": "Admin User",
			"hasLiveVersion": true,
			"modUser": "dotcms.org.1"
		},
		"reply": []
	}
]
```

**Comment by bryanboza** [comment]
Fixed, tested on release 21.06 // Postgres // FF
